### PR TITLE
Add users 1

### PR DESCRIPTION
**Pull request by BoluK001**

### CODE REVIEW

File: users.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
 <html>
     <head>
+
```
Comment (AMSDevCenter) [inline users.html:4]: There are duplicate lines of code

**Review by AMSDevCenter — CHANGES_REQUESTED** [review]
Left comments

File: users.html (hunk 1 above)
Comment (BoluK001) [reply users.html:4]: fixed

**Review by AMSDevCenter — APPROVED** [review]
LGTM

**Pull request merged**